src/document-click.js

```diff
@@ -108,6 +113,21 @@ export function onDocumentElementClick_(e, viewport) {
   const win = doc.defaultView;
 
   const tgtLoc = parseUrl(target.href);
+
+  // On Safari iOS, custom protocol links will fail to open apps when the
+  // document is iframed - in order to go around this, we set the top.location
+  // to the custom protocol href.
+  const isSafariIOS = platform.isIos() && platform.isSafari();
+  const isEmbedded = win.parent && win.parent != win;
+  const isCustomProtocolHref = (
```

Comment:
It'd be easier to test for `isNormalProtocol`, then `!` in the conditional below:

```js
const isNormalProtocol = /^https?:$/.test(tgtLoc.protocol)
if (isSafari && isEmbedded && !normalProtocol) {
  //...
}
```